__tests__/app.test.js

```diff
@@ -83,4 +80,36 @@ describe("API testing", () => {
         });
     });
   });
+  describe("/api/articles/:article_id", () => {
+    test("return correct article when passed an article id ", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.articles).toHaveLength(1);
+          body.articles.forEach((article) => {
+            expect(article).toEqual(
+              expect.objectContaining({
+                article_id: expect.any(Number),
+                article_img_url: expect.any(String),
+                author: expect.any(String),
+                body: expect.any(String),
+                created_at: expect.any(String),
+                title: expect.any(String),
+                topic: expect.any(String),
+                votes: expect.any(Number),
+              })
+            );
+          });
+        });
+    });
```

Comment:
Some nice testing here, however as we're only expecting to get the information back from a single article we can add the literal object that we expect to receive back rather than looping through an array that contains a single item. This would also mean that you could return rows [0] instead of rows